package.json

```diff
@@ -1,15 +1,14 @@
 {
-  "name": "bull_exporter",
-  "version": "1.3.9",
-  "description": "Promethus Exporter for Bull Queues",
+  "name": "bullmq_exporter",
+  "version": "1.0.0",
+  "description": "Promethus Exporter for BullMQ Queues",
   "main": "dist/src/index.js",
-  "repository": "git@github.com:UpHabit/bull_exporter.git",
-  "author": "Gabriel Castro <GabrielCastro@users.noreply.github.com>",
+  "repository": "git@github.com:marius-juro/bullmq_exporter.git",
   "license": "MIT",
   "scripts": {
     "build": "tsc -p .",
     "lint": "tslint -p .",
-    "test": "jest --ci"
+    "test": "jest --ci --detectOpenHandles"
```

Comment:
jest docs on `--detectOpenHandles`: "This option has a significant performance penalty and should only be used for debugging."

even though there aren't a lot of tests here, it seems like bad practice to use debugging flags by default

I am curious why jest takes more than 1s to terminate, but as long as we know there are no open handles I don't think this is needed